### PR TITLE
[TIMOB-19081] Android: Add Theme.AppCompat.NoTitleBar to default themes

### PR DESCRIPTION
**Pull request by FokkeZB**
https://jira.appcelerator.org/browse/TIMOB-19081

### CODE REVIEW

File: android/templates/build/theme.xml [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
 <resources>
-	<style name="Theme.AppCompat.Fullscreen">
+	<style name="Theme.AppCompat.NoTitleBar">
 		<item name="android:windowNoTitle">true</item>
-		<item name="android:windowFullscreen">true</item>
 		<item name="android:windowContentOverlay">@null</item>
 		<item name="android:windowActionBar">false</item>
 		<!-- AppCompat Compatibility -->
 		<item name="windowActionBar">false</item>
-		<item name="windowNoTitle">true</item>
```
Comment (ashcoding) [inline android/templates/build/theme.xml]: Just wondering, why is this removed? "windowNoTitle"

Comment (FokkeZB) [inline android/templates/build/theme.xml]: You were saying? ;)

**Comment by ashcoding** [comment]
Code reviewed. Functionally tested with test App, working. :+1:
Sorry to nitpick but, could you squash it into 1 commit?

**Comment by FokkeZB** [comment]
OK, made a mess :santa:

**Comment by ashcoding** [comment]
I find http://makandracards.com/makandra/527-squash-several-git-commits-into-a-single-commit to be really useful (Easy mode). Followed by a --force push.

**Comment by FokkeZB** [comment]
Closing for:
https://github.com/appcelerator/titanium_mobile/pull/6966

**Comment by FokkeZB** [comment]
Thanks @ashcoding, went ahead and created a new one ;)

**Comment by ashcoding** [comment]
Okay!
